propagation/usecases.md

```diff
@@ -0,0 +1,167 @@
+
+### Examples
+
+* The project has created an EO data processing pipeline for generating a new air quality index and wants to encapsulate
+  it in a UDP or OGC Application Package to facilitate easy execution by stakeholders and
+  integration into a dashboard.
+
+### Solution
+
+APEx offers a set of initial [guidelines](../interoperability/algohosting.md) and
+online [resources](index.qmd#how-to-build-an-on-demand-service)
+to help projects with integrating their processing workflow into an APEx-compliant technology. Should additional
+assistance be required, the [APEx Algorithm Porting](porting.md) service can provide further support. After an initial
+analysis, APEx experts will propose an appropriate APEx-compliant technology and assist your team in its implementation.
+
+:::{.callout-tip}
+
+# Want to share your algorithm with the APEx community?
+
+Please refer to Use Case 4 to learn how APEx can support you in sharing your algorithm with the community.
+:::
+
+### Outcome
+
+The algorithm is successfully transformed and is now available as an openEO UDP or OGC Application Package.
+
+## Use Case 4: Make your algorithm available in the APEx Algorithm Catalogue
+
+### Scenarios
+
+* A project has implemented its algorithm as an openEO User Defined Process (UDP) or packaged it as an OGC Application
+  Package and wants to make it available to the APEx community.
+
+::: {.callout-tip}
+
+# Your algorithm is not yet available as an openEO UDP or packaged as an OGC Application Package?
+
+If your algorithm has not yet been implemented in an APEx-compliant technology, please refer to the previous use case to
+discover how APEx can provide support.
+:::
+
+### Examples
+
+* The project has created an openEO UDP or OGC Application Package that calculates a new air quality index and wants to
+  share this service with the EO community.
+* The project has developed a new machine learning model encapsulated in an openEO UDP or Application Package and wants
+  to ensure it runs stably and within a controlled cost range.
+* The project has developed an OGC Application Package and is seeking support to host it on an APEx-compliant platform.
+
+### Solution
+
+The [APEx Algorithm Onboarding](onboarding.md) service offers comprehensive support for integrating your algorithm into
+the APEx Algorithm Catalogue, ensuring its availability for reuse within the EO community. The service includes:
+
+* Assistance in selecting an APEx-compliant algorithm hosting platform to guarantee that your algorithm is available for
+  execution.
+* Provision of tools for automated testing and benchmarking of your algorithm.
+* Support and resources for registering your algorithm in the APEx Algorithm Catalogue.
+
+### Outcome
+
+The algorithm is available in the APEx Algorithm Catalogue, enhancing its visibility and usability within
+the EO community as an APEx-compliant service.
+
+## Use Case 5: Automating EO Data Processing Pipelines for Large-Scale Data Processing
```

Comment:
I suggest to not introduce 'pipeline' in our vocabulary, because it's not a well-defined term.
APEx is about on-demand services, and the upscaling service is very much based on them.
Other sorts of 'EO pipelines' for sure exist, but we don't provide tooling to upscale them. (Projects that wish to interact at the level of docker containers for instance can just go ahead and do that with off the shelve open source tools.)

```suggestion
## Use Case 5: Generating large scale maps or results based on on-demand EO services
```